CHANGES.md

```diff
@@ -58,7 +58,12 @@
 * FfmpegProcessor, KakaduNativeProcessor, OpenJpegProcessor, and
   PdfBoxProcessor use libjpeg-turbo to write JPEGs if it is available.
 * Added the `scaled` position for image overlays.
-* Updated Apache PDFBox to version 2.0.21.
+* Updated Apache PDFBox to version 2.0.22.
+* PdfBoxProcessor properties added to better control memory
+    * `processor.pdf.scratch_file_enabled` allows use of a scratch file
+    * `processor.pdf.max_memory_bytes` allows control over how much memory a PDF thread consumes (-1 means unlimited)
+    * `processor.pdf.max_storage_bytes` allows control over how much total bytes are used to render a PDF image (memory + temp file)
+    * `processor.pdf.scratch_location` allows customization of path to PDF scratch file
```

Comment:
Could `temp_path` be used in place of this?

Reply:
You are absolutely correct.  I over-thought this because the scratch-file out-grew my system temp location but for some reason didn't think to just update the system temp location  I'll remove this setting and use `temp_path`